test/functionalTest/cases/0000_cli/bool_option_with_value.test

```diff
@@ -85,5 +85,6 @@ Usage: contextBroker  [option '-U' (extended usage)]
                       [option '-strictNgsiv1Ids' (additional checks for id fields in the NGSIv1 API)]
                       [option '-disableCustomNotifications' (disable NGSIv2 custom notifications)]
                       [option '-logForHumans' (human readible log to screen)]
+                      [option '-metrics' (turn off the 'metrics' feature - to gain efficiency)]
```

Comment:
I'd suggest to the message short (i.e. just "(turn off the 'metrics' feature)"). Considerations regarding how this affects performance should be done in other place (in particular, in the perf_tunning.md document).